apps/src/applab/applab.js

```diff
@@ -873,7 +892,8 @@ function setupReduxSubscribers(store) {
       (isDataMode && view !== lastView) ||
       changedToDataMode(state, lastState)
     ) {
-      onDataViewChange(
+      loadDataForView(
```

Comment:
onDataViewChange has been renamed to `loadDataForView` and refactored into `loadDataView.js`. This name better reflects the usage in Datablock Storage in particular, but also gives a better sense of what the firebase code does.